packages/og-image/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@resvg/resvg-wasm": "2.6.2",
-    "@theguild/components": "workspace:*",
+    "@theguild/components": "^6",
```

Comment:
should use from workspace

Reply:
oh damn, I forgot about it. We need to fix `og-image` to render nicely with new icons, that's why I locked it to ^6.

Reply:
lgrm